### ISSUE TITLE
BC break with PHP 5.5.9

### ISSUE THREAD

**Issue by Sacrileigous** [hechoendrupal/drupal-console-core#274]
drupal-console: PHP 5.5.9 or higher is required to use the Drupal Console application.

Commit #267  breaks BC with PHP 5.5.9 because dirname optional levels parameter is introduced only in PHP 7.

```Warning: dirname() expects exactly 1 parameter, 2 given in /drupal/vendor/drupal/console-core/src/Utils/ConfigurationManager.php on line 214

Notice: Undefined index: config in /drupal/vendor/drupal/console-core/src/Utils/ConfigurationManager.php on line 67

Catchable fatal error: Argument 1 passed to Dflydev\DotAccessConfiguration\YamlFileConfigurationBuilder::__construct() must be of the type array, null given, called in /drupal/vendor/drupal/console-core/src/Utils/ConfigurationManager.php on line 68 and defined in /drupal/vendor/dflydev/dot-access-configuration/src/Dflydev/DotAccessConfiguration/YamlFileConfigurationBuilder.php on line 31

**Comment by lukasvs** [comment]
Also problem!

**Comment by lukasvs** [comment]
How can I quickly start the console again?

**Comment by Sacrileigous** [comment]
@lukasvs downgrade drupal-console to 1.3.2 or set your PHP version to at least 7.0.

**Comment by jmolivas** [comment]
My bad fxing in a few

**Comment by jmolivas** [comment]
Fix PR https://github.com/hechoendrupal/drupal-console-core/pull/277

**Comment by jmolivas** [comment]
@Sacrileigous @lukasvs This should be fixed on 1.5.0 could you update to latest version and confirm.

**Comment by itsmeguru786** [comment]
Thanks! Raising to version 7 helped resolve this for me.
